### PR TITLE
doc: Add info about supported format for URI

### PR DESCRIPTION
**Pull request by ghost**
+ Bitcoin Core only supports valid bitcoin `address` for URI and nothing else
+ Only first example will work from BIP 21 and others will give error (Check PR: https://github.com/bitcoin-core/gui/pull/280 or https://github.com/bitcoin/bitcoin/pull/21755 for new error messages being discussed)

### CODE REVIEW

**Comment by prayank23** [comment]
PR #176 mentioned for BIP 21 has no commits so maybe we can also add/replace it with https://github.com/bitcoin/bitcoin/commit/70f55355e29c8e45b607e782c5d76609d23cc858

File: doc/bips.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ BIPs that are implemented by Bitcoin Core (up-to-date up to **v22.0**):
...
 * [`BIP 14`](https://github.com/bitcoin/bips/blob/master/bip-0014.mediawiki): The subversion string is being used as User Agent since **v0.6.0** ([PR #669](https://github.com/bitcoin/bitcoin/pull/669)).
 * [`BIP 16`](https://github.com/bitcoin/bips/blob/master/bip-0016.mediawiki): The pay-to-script-hash evaluation rules have been implemented since **v0.6.0**, and took effect on *April 1st 2012* ([PR #748](https://github.com/bitcoin/bitcoin/pull/748)).
-* [`BIP 21`](https://github.com/bitcoin/bips/blob/master/bip-0021.mediawiki): The URI format for Bitcoin payments has been implemented since **v0.6.0** ([PR #176](https://github.com/bitcoin/bitcoin/pull/176)).
+* [`BIP 21`](https://github.com/bitcoin/bips/blob/master/bip-0021.mediawiki): The URI format for Bitcoin payments has been implemented since **v0.6.0** ([PR #176](https://github.com/bitcoin/bitcoin/pull/176)). Only address is supported in URI as mentioned in first [example](https://github.com/bitcoin/bips/blob/master/bip-0021.mediawiki#examples).
```
Comment (Rspigler) [inline doc/bips.md:8]: nit: "Only [the] address is supported in [the] URI as mentioned in [the] first example"

Comment (prayank23) [reply doc/bips.md:8]: "[the] address" or "a address" or "a Bitcoin address"?

**Comment by Rspigler** [comment]
I tested here: https://github.com/bitcoin-core/gui/pull/280#issuecomment-830664559 and a label was still present for a URI with a valid address

**Comment by prayank23** [comment]
> I tested here: bitcoin-core/gui#280 (comment) and a label was still present for a URI with a valid address

I just tried all examples in v0.21.1 and they are working. Will close this PR and respond in https://github.com/bitcoin-core/gui/pull/280 after testing.

**Pull request closed without merging**